web/frontend/src/components/slam.vue

```diff
@@ -17,16 +22,24 @@ const props = defineProps<{
   client: Client
 }>();
 
+const operationID = 'OP-ID'
 const selected2dValue = $ref('manual');
 const selected3dValue = $ref('manual');
 let pointCloudUpdateCount = $ref(0);
 let pointcloud = $ref<Uint8Array | undefined>();
 let pose = $ref<commonApi.Pose | undefined>();
 let show2d = $ref(false);
 let show3d = $ref(false);
+let showAxes = $ref(true);
 let refresh2DCancelled = true;
 let refresh3DCancelled = true;
+let updatedDest = $ref(false);
+let destinationMarker = $ref(new THREE.Vector3());
+let moveClick = $ref(true);
+let basePose = new commonApi.Pose()
+const motionServiceReq = new motionApi.MoveOnMapRequest();
 
+const allowMove = $computed(() => window.localStorage.getItem(operationID) === null)
```

Comment:
Is there any documentation on how the UI should integrate with the opId system?

Reply:
i am not aware of any existing documentation. I also am unaware of any existing opID system.

using `window.localStorage` is a hack at mimicking that behavior.

Reply:
Who owns the OpId system? Can you ensure they approve of this approach / sign off on this PR?